backend-nodejs/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.9"
```

Comment:
Again, persistence probably isn't necessary, so I wouldn't bother having the PostgreSQL database spun up here